eclipse-collections-code-generator/src/main/resources/impl/stream/primitive/primitiveSingletonSpliterator.stg

```diff
@@ -0,0 +1,69 @@
+import "copyrightAndOthers.stg"
+
+forSpecializedStream() ::= "true"
+
+targetPath() ::= "org/eclipse/collections/impl/stream/primitive"
+
+fileName(primitive) ::= "<primitive.name>SingletonSpliterator"
+
+
+class(primitive) ::= <<
+<body(primitive.type, primitive.name)>
+>>
+
+body(type, name) ::= <<
+<copyrightAndOthers()>
+
+package org.eclipse.collections.impl.stream.primitive;
+
+import java.util.Spliterator;
+import java.util.function.<name>Consumer;
+
+/**
+ * <name>SingletonSpliterator is a specialized spliterator of type <type>.
+ * This file was automatically generated from template file primitiveSingletonSpliterator.stg.
+ *
+ * @since ...
```

Comment:
Since 10.0, right @nikhilnanivadekar ?

Reply:
are there additional places where u should put this @since tag? should i put it in every function?

Reply:
I am fine with just at the top level as long as the javadoc shows it was added in 10.0
@motlin is right it is since 10.0

Reply:
I am fine with just at the top level as long as the javadoc shows it was added in 10.0
@motlin is right it is since 10.0